benchmark/compare.js

```diff
@@ -76,11 +77,9 @@ if (showProgress) {
       // Construct configuration string, " A=a, B=b, ..."
       let conf = '';
       for (const key of Object.keys(data.conf)) {
-        conf += ` ${key}=${JSON.stringify(data.conf[key])}`;
+        conf += ` ${key}=${inspect(data.conf[key])}`;
```

Comment:
I'm actually wondering, is there a place where we benefit from using `inspect` here as opposed to simply `String(data.conf[key])`? Because as far as I can tell, `conf` is generally number, string or true/false.

Reply:
We do because we will actually be able to distinguish the data types by using `util.inspect`. By only converting it to a string, the string `'true'` would look the same as the boolean `true`.

Reply:
That's fair.

Personally, I'm not sure I actually love that part of the functionality but obviously that's not something that needs to be addressed in this PR. Since it's generating CSV, having true and "true" be equivalent is almost preferable to me.

One other thing... `util.inspect` will do the following:
```
util.inspect("true");
// '\'true\''
JSON.stringify("true");
// "true"
```

The former doesn't seem correct for the purposes of CSV.

Reply:
FWIW one of the problems with single quotes is there seems to be no accepted way of handling them. Some parsers seem to treat them as equivalent to double quotes (meaning they need to be escaped) and others treat them as just a character.

Reply:
@apapirovski I wrote this with the RFC 4180 specs in mind. If a parser does something else (and many do), then that is just tough luck. Creating CSV files that work in all parsers is a very big challenge. We just need to make sure it works with `read.csv` in `R`.